installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -75,3 +75,29 @@ include::modules/install-sno-installing-with-usb-media.adoc[leveloffset=+1]
 include::modules/install-booting-from-an-iso-over-http-redfish.adoc[leveloffset=+1]
 
 include::modules/creating-custom-live-rhcos-iso.adoc[leveloffset=+1]
+
+== Installing {sno} with {ibmzProductName} and {linuxoneProductName}
+
+Installing a single node cluster on {ibmzProductName} and {linuxoneProductName} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibmzProductName} and {linuxoneProductName}" or the "Installing a cluster with z/VM on {ibmzProductName} and {linuxoneProductName}" procedure.
+
+[discrete]
+=== Hardware requirements
```

Comment:
@chidanandbh Can you please verify if this is correct. @holgwolf asked me to add Hardware Requirements